### ISSUE TITLE
Code Samples / Docs

### ISSUE THREAD

**Issue by JohnFulsom** [MicrosoftDocs/winrt-api#1117]
Hi, I'd like to know if there are any code samples or deeper documentation for this API, or if there will be any forthcoming any time soon. Thanks.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 11017ce2-ced1-2f44-0a34-c4013cedfd27
* Version Independent ID: 067fba70-52d7-65e1-8f8d-790db6012efe
* Content: [Windows.Devices.Display.Core Namespace - Windows UWP applications](https://docs.microsoft.com/en-us/uwp/api/windows.devices.display.core#feedback)
* Content Source: [https://cpubwin.visualstudio.com/DefaultCollection/windows-uwp/_git/winrt-api-build?path=%2fwinrt-api-build%2fwindows.devices.display.core.yml&version=GBlive&_a=contents](https://cpubwin.visualstudio.com/DefaultCollection/windows-uwp/_git/winrt-api-build?path=%2fwinrt-api-build%2fwindows.devices.display.core.yml&version=GBlive&_a=contents)
* Product: **uwp**
* Technology: **winrt-reference**
* Git Login: **eliotcowley**
* Microsoft Alias: **elcowle**

**Comment by OlrikSato7** [comment]
Hi. Just a message to say that JohnFulsom would not be the only one interested.

**Comment by englishb** [comment]
Yes, I would also be quite interested.

**Comment by BattleAxeVR** [comment]
+1

**Comment by Gavin-Williams** [comment]
Please can we get some discussion about what this is.

**Comment by rpavlik** [comment]
This is the furthest I've gotten with this API: I've included some logs, and questions in the comments. https://gitlab.collabora.com/rpavlik/windows-display-enum

**Comment by stevewhims** [comment]
Hi, @JohnFulsom (and others), thanks for filing this issue, and sorry for the delay in responding. At the top of this topic, I've added a link to a sample application. We also recently added more API reference topics, filling some gaps.

As for more in-depth conceptual content, I'm not aware of any that's planned, or awaiting publication. However, I'll raise awareness of this thread with the feature team.

Thanks!
Steve
